docs/guide/test-fixtures.md

```diff
@@ -154,7 +154,7 @@ You may also assign an alias to a fixture. In the above example, the `UserProfil
 In the test methods, you may then access a fixture object using its alias in `grabFixture()` method. For example,
 
 ```php
-$profile = $I->grabFixture('profiles', 'user1');
+$profile = $I->grabFixture('profiles');
```

Comment:
I think it was correct. `user1` is the index in the fixtures array so there we are grabbing a profile for user 1.

Reply:
Well, check the full paragraph:

> For example,
>     $profile = $I->grabFixture('profiles', 'user1');
> will return the UserProfileFixture object.

That's not correct IMO, because you get a `User` instance and not the `UserProfileFixture` itself if you call it with the `user1` argument, don't you?

Reply:
Yes, you are correct. Change is good.